README.md

```diff
@@ -102,6 +102,7 @@ Releases
 
 - Change absolute `@see` links to relative paths,
   so we're not assuming a root URL for the styleguide.
+- Render type `html` examples to HTML.
```

Comment:
It looks like 0.5.3 has already been released, so this should all be updated to match the missing changes.